components/globals/AdminNav.tsx

```diff
@@ -47,12 +47,11 @@ const AdminNav = (props: AdminNavProps): React.ReactElement => {
             <Link href="/admin/view-templates">{t("adminNav.templates")}</Link>
           </li>
         )}
-        {ability?.can("view", "Flag") &&
-          ability?.can("update", "FormRecord", "[publishingStatus]") && (
-            <li className="gc-horizontal-item">
-              <Link href="/admin/flags">{t("adminNav.features")}</Link>
-            </li>
-          )}
+        {ability?.can("view", "Flag") && ability?.can("update", "FormRecord", "isPublished") && (
+          <li className="gc-horizontal-item">
```

Comment:
This was probably a mistake from me in an ealier PR around permissions.
I don't think we need to check if a user has publishing permissions to access the Application Settings page.